README.md

```diff
@@ -145,6 +169,120 @@ such as Bloc and Provider
 </br>
 </br>
 
+## Design Guidelines
+
+This section outlines the design guidelines for the modal sheet component, including breakpoints, safe areas, scroll logic, and behavior across different devices. The guidelines are based on the Wolt design system specs.
+
+### Overview
+
+The modal sheet component is used to display critical information or interactive elements to users in a non-intrusive way. This component adapts to different screen sizes and orientations, ensuring a consistent and accessible user experience.
+
+### Breakpoints
+
+The modal sheet component adjusts its layout based on the following breakpoints:
+
+- **Breakpoint Large**: Width ≥ 1400px
+- **Breakpoint Medium**: 768px ≤ Width < 1400px
+- **Breakpoint Small**: 524px ≤ Width < 768px
+- **Breakpoint XSmall**: Width < 524px
+
+<img width="1158" alt="Screenshot 2024-06-30 at 17 15 49" src="https://github.com/yarneo/wolt_modal_sheet/assets/4066863/7dd39e47-8045-45a5-b03b-c62da805bc5f">
+
+#### Modal Types
+
+- **Alert**: Used for critical information that requires immediate attention.
+- **Dialog**: Used for single user actions or to convey information related to changes in state (success, errors).
+- **Side Sheet**: Used to focus users' attention on a specific task while keeping the context visible.
+- **Bottom Sheet**: Used for additional options or actions.
+- **Full Bottom Sheet**: Used to present content that requires full screen height.
+
+### Safe Areas
```

Comment:
We can remove this part since we are delegating the safe area metrics and value calculations to Flutter.

Reply:
done